File: 17s2/test11/devowel_file.pl
#!/usr/bin/perl -w

$target_file = $ARGV[0];

open my $f, $target_file or die;

while (my $line = <$f>) {
	$line =~ s/[aeiouAEIOU]//g;
	push @line_arr, $line;
	#print("$line");
}
close $f;

open("RESULT", ">$target_file");
foreach $item (@line_arr) {
	print RESULT "$item";
}

File: 17s2/test11/devowel_file.py
#!/usr/bin/python3

import sys,re

file_name = sys.argv[1]
to_be_written_list = []
with open(file_name, "r") as file1:
    for line in file1.readlines():
        stri=""
        for char in line:
            if re.match(r'[^aeiouAEIOU]',char):
                stri += char;
        to_be_written_list.append(stri)

with open(file_name, "w") as file2:
    for adjust_line in to_be_written_list:
        file2.write(adjust_line)
        
